From 5c8566bd998cc62483091ef2aae2c124b4857435 Mon Sep 17 00:00:00 2001
From: Junichi Uekawa <dancer@debian.org>
Date: Sun, 30 Apr 2023 16:33:16 +0900
Subject: [PATCH] Remove windows dependencies.

Debian doesn't build for Windows, and this optional dependency is
never used.
---
 Cargo.toml | 4 ----
 1 file changed, 4 deletions(-)

Index: named-lock/Cargo.toml
===================================================================
--- named-lock.orig/Cargo.toml
+++ named-lock/Cargo.toml
@@ -49,16 +49,3 @@ features = ["v4"]
 
 [target."cfg(unix)".dependencies.libc]
 version = "0.2.132"
-
-[target."cfg(windows)".dependencies.widestring]
-version = "1.0.2"
-
-[target."cfg(windows)".dependencies.winapi]
-version = "0.3.9"
-features = [
-    "handleapi",
-    "synchapi",
-    "winbase",
-    "winnt",
-    "winerror",
-]
